contracts/gamecontract/Gamecontract.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+contract GameContract is Ownable {
+    error InvalidGameContractUser();
+
+    mapping(address => bool) private signerMap;
+
+    uint256 private totalUserGameContract;
```

Comment:
It must be `signerTotal`, you're really bad at naming stuff.

Reply:
have check